docs/sources/administration/image_rendering.md

```diff
@@ -191,7 +191,9 @@ t=2019-12-04T12:39:22+0000 lvl=error msg="Request Completed" logger=context user
 
 If this happens, then you have to add the certificate to the trust store. If you have the certificate file for the internal root CA in the file `internal-root-ca.crt.pem`, then use these commands to create a user specific NSS trust store for the Grafana user (`grafana` for the purpose of this example) and execute the following steps:
 
-```[root@server ~]# [ -d /usr/share/grafana/.pki/nssdb ] || mkdir -p /usr/share/grafana/.pki/nssdb
+```
+[root@server ~]# apt install -y libnss3-tools
```

Comment:
Good catch for the missing line feed. Also agree that `libnss3-tools` is required to have `certutil`.
I'm not sure I'd put the apt install line though, since apt is really tide to the debian world. Maybe we could simply add a line stating it needs to be installed for the following commands to work? WDYT?

Reply:
Yes, leave out Debian-specific syntax.

Reply:
I think it's located in `nss-tools` on in the RHEL/Fedora/CentOS family.